test/controller.bar.tests.js

```diff
@@ -487,4 +489,154 @@ describe('Bar controller tests', function() {
 		expect(bar._model.borderColor).toBe('rgb(0, 255, 0)');
 		expect(bar._model.borderWidth).toBe(1.5);
 	});
+
+	describe('Bar width', function() {
+		beforeEach(function() {
+			// 2 datasets
+			this.data = {
+				labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+				datasets: [{
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}, {
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}]
+			};
+		});
+
+		afterEach(function() {
```

Comment:
I think @simonbrunel meant that he'd prefer the `expect` lines to not be inside the `afterEach` method and instead be in the `it` method